components/DefinedTerm.js

```diff
@@ -63,7 +63,8 @@ const TranslatedDefinitions = defineMessages({
   },
   [Terms.ESTIMATED_BUDGET]: {
     id: 'CollectivePage.SectionBudget.Annual.Definition',
-    defaultMessage: 'Projected annual budget based on total financial contributions from the past 12 months.',
+    defaultMessage:
+      'Projected annual budget based on total financial contributions from the past 12 months. Monthly recurring: {monthlyRecurring}. Total received in the last 12 months: {totalReceivedInLast12Months}',
```

Comment:
Is there any way we can format this to make it easier to quickly parse the information?

Maybe something like

```
Projected annual budget based on total financial contributions from the past 12 months. 

Monthly recurring: {monthlyRecurring}
Total received in the last 12 months: {totalReceivedInLast12Months}
```

Reply:
Sure thing. I will take a look at how to do this